### ISSUE TITLE
Sluggish pupillary reaction

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7358]
**Preferred term label:**
Sluggish pupillary reaction

**Synonyms**

**Definition (free text, please give PubMed ID)**
also other terms from here https://www.ncbi.nlm.nih.gov/books/NBK537180/

he speed of pupillary
reactivity is recorded as brisk, sluggish, or nonreactive.
Normally, pupils should constrict briskly in response
to light. A sluggish or slow pupillary response may
indicate increased ICP, and nonreactive pupils are
often associated with severe increases in ICP and/or
severe brain damage

Adie’s tonic pupil 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
see https://pubmed.ncbi.nlm.nih.gov/17975799/   OMIM:267750 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
THis was, Slow pupillary light response HP:0030211, adding synonym
